### ISSUE TITLE
Deplotment?

### ISSUE THREAD

**Issue by umarfarooq360** [codeforpakistan/Report-A-Rishwat#4]
Has this app been deployed anywhere? I also didn't see this exhibited on the codeforpakistan website?

**Comment by Snake2k** [comment]
Not really, we deployed it on a temporary server for testing. But that's it.
